include/sdf/Box.hh

```diff
@@ -65,6 +68,9 @@ namespace sdf
     /// \return A reference to a gz::math::Boxd object.
     public: gz::math::Boxd &Shape();
 
+    /// \brief Calculate an return the Mass Matrix values for the Box
+    public: std::optional< gz::math::MassMatrix3d >  MassMatrix(const double density);
```

Comment:
```suggestion
    \param[in] _density ...
    \return ...
    public: std::optional<gz::math::MassMatrix3d>  MassMatrix(const double _density);
```

Reply:
Corrected in b092904db4ef5c491672619076e75f4509b3369b